src/hotspot/share/opto/memnode.cpp

```diff
@@ -1548,11 +1549,14 @@ static bool stable_phi(PhiNode* phi, PhaseGVN *phase) {
 // Phi *base*. This method is essentially a copy of the validations performed
 // by 'split_through_phi'. The first use of this method was in EA code as part
 // of simplification of allocation merges.
+// Some differences from original method:
```

Comment:
What "original method" do you refer to here?

Reply:
The current method, `can_split_through_phi_base`, is just a helper that contains a copy of the validations performed by `split_through_phi`. The original method is `split_through_phi`.